keepercommander/commands/record.py

```diff
@@ -668,12 +668,11 @@ def execute(self, params, **kwargs):
 
         files = []
         if 'file' in kwargs:
-            for name in kwargs['file']:
```

Comment:
'file' parameter is defined as
`upload_parser.add_argument('--file', dest='file', action='append', required=True, help='file name to upload.')`
So, file parameter is expected to be a list of strings. 
Probably, 'argparse' package for some Python versions on Windows do not parse a single 'file' parameter as a list.
The proper solution would be checking type of kwargs['file'] and run the old code if it's a list or new code if it's a str

Reply:
What Python version do you use?

Reply:
Makes sense...
I'm uisng the python version which is included in WinPy-3670 which is python-3.6.7.amd64

Reply:
I cannot duplicate this issue in WinPy-3670.
`upload-attachment --file="C:\\Users\sk\aa" NmMRFMQOU5tHJMQXtMQRYL`
command successfully uploads file attachment.
Is it possible that system 'argparse.py' script is altered or conflicting package is installed with pip?
It would be nice to know the problem before applying the patch.

Reply:
@sk-keeper Thanks for taking your time to investigate this so called problem I encountered. I've now found what the issue was from my part. As I'm creating a wrapper for powershell there was one of the jumps between systems that made the input wrong. As you mentioned in the first part dest='file' was what was lacking in my script and therefore it failed. 

You could remove this pull-request as it is not needed if you do it the right way.